src/Mvc/Cli/Commands/Data/DumpCommand.php

```
@@ -0,0 +1,500 @@
+			return 1;
+		}
+	}
+
+	/**
+	 * Get list of tables that will be exported
+	 *
+	 * @param DataExporter $exporter
+	 * @param array $exportOptions
+	 * @return array
+	 */
+	private function getTableList( DataExporter $exporter, array $exportOptions ): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$exportOptions'. (undefined)

(UnusedFormalParameter)